build/__phpVersions.sh

```diff
@@ -1,20 +1,20 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-PHP_RUNTIME_BASE_TAG='20220803.3'
-PHP_FPM_RUNTIME_BASE_TAG='20220817.1'
+PHP_RUNTIME_BASE_TAG='20220827.1'
+PHP_FPM_RUNTIME_BASE_TAG='20220827.1'
 COMPOSER_VERSION='2.0.8'
 COMPOSER_SETUP_SHA384='55ce33d7678c5a611085589f1f3ddf8b3c52d662cd01d4ba75c0ee0459970c2200a51f492d557530c71c15d8dba01eae'
 COMPOSER1_9_VERSION='1.9.3'
 COMPOSER1_10_VERSION='1.10.19'
 COMPOSER2_0_VERSION='2.0.8'
 COMPOSER2_2_VERSION='2.2.9'
 COMPOSER2_3_VERSION='2.3.4'
-PHP81_VERSION='8.1.6'
+PHP81_VERSION='8.1.9'
```

Comment:
@qianz2 should we update our integration tests for these new versions now?

Reply:
@william-msft it should be updated automatically with the tests referencing the `Php81Version` variable, correct?

Reply:
@william-msft I checked the integration tests, they're referring to the major minor version constants so I think it's updated.

Reply:
Ah yes, since it's a minor version bump.